azext_hanaonazure/_params.py

```diff
@@ -11,4 +11,19 @@
 def load_arguments(self, _):
     with self.argument_context('hanainstance') as c:
         c.argument('resource_group_name', arg_type=resource_group_name_type)
-        c.argument('instance_name', options_list=['--instance-name', '-n'], help="The name of the SAP HANA instance", id_part='name')
\ No newline at end of file
+        c.argument('instance_name', options_list=[
+                   '--instance-name', '-n'], help="The name of the SAP HANA instance", id_part='name')
+        c.argument('hana_vnet', options_list=[
```

Comment:
What happens if the user adds this parameters for other commands?

Reply:
nothing happens in that case. It only shows up for this command

Reply:
I did add an argument context, so now they will definitely only show up under `hanainstance enable-monitoring`

Reply:
I still think it will be cleaner to have a `monitor` sub-command. Then have the arguments scoped to that sub-command. See az network nsg/dns/...